hadoop-ozone/ozone-manager/src/main/java/org/apache/hadoop/ozone/protocolPB/OzoneManagerRequestHandler.java

```diff
@@ -385,9 +386,22 @@ public OMResponse handleReadRequest(OMRequest request) {
     return responseBuilder.build();
   }
 
+  @VisibleForTesting
+  public static int handle_write_sleep = 0;
+
   @Override
   public OMClientResponse handleWriteRequest(OMRequest omRequest,
       long transactionLogIndex) throws IOException {
+    
+    if (handle_write_sleep > 0) {
+      // Only for test purpose
+      try {
+        Thread.sleep(handle_write_sleep);
+      } catch (InterruptedException e) {
+        Thread.currentThread().interrupt();
+      }
+    }
```

Comment:
How about use `CustomFaultInjector` extends `FaultInjector` here? It is similar to `SnapshotPauseInjector` (or XXXFaultInjector in HDFS).